### PR TITLE
docs infra: reduce collaborator image sizes

### LINKED ISSUE #41287: ci(docs-infra): reduce sizes of all contributor images to be below 30KB
PR #41253 added a new check to verify that the contributor images (shown in https://angular.io/about) do not exceed 30KB in size. These images are always displayed as [168px x 168px][1], so there is no benefit in serving larger images. Additionally, these images are cached by the ServiceWorker (as part of the [docs-lazy][2] asset group) once a user visits https://angular.io/about, so that is another reason to be mindful of their size.

However, some of the existing images exceeded the newly imposed size limit. These have been added to a hard-coded [exception list][3] for now.

Eventually, we should reduce the sizes of these images to be below the limit and remove the exception list altogether.
(This issue is for tracking this work.)

Images that need to be downsized (in alphabetic order):
- [ ] [ahsanayaz.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/ahsanayaz.jpg) (77.7KB)
- [ ] [alainchautard.png](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/alainchautard.png) (342.8KB)
- [ ] [alan-agius4.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/alan-agius4.jpg) (31.2KB)
- [ ] [andrew-kushnir.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/andrew-kushnir.jpg) (68.9KB)
- [ ] [brian-love.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/brian-love.jpg) (162.2KB)
- [ ] [cexbrayat.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/cexbrayat.jpg) (33.9KB)
- [ ] [christianliebel.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/christianliebel.jpg) (42.9KB)
- [ ] [gerardsans.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/gerardsans.jpg) (88.8KB)
- [ ] [jessicajaniuk.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/jessicajaniuk.jpg) (33.7KB)
- [ ] [JiaLiPassion.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/JiaLiPassion.jpg) (38.5KB)
- [ ] [juristr.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/juristr.jpg) (207.2KB)
- [ ] [katerina.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/katerina.jpg) (58.8KB)
- [ ] [kimmaida.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/kimmaida.jpg) (76.0KB)
- [ ] [kyliau.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/kyliau.jpg) (35.4KB)
- [ ] [lacolaco.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/lacolaco.jpg) (151.0KB)
- [ ] [leonardo.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/leonardo.jpg) (31.6KB)
- [ ] [nirkaufman.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/nirkaufman.jpg) (34.4KB)
- [ ] [patovargas.png](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/patovargas.png) (82.1KB)
- [ ] [sajee.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/sajee.jpg) (56.4KB)
- [ ] [sonukapoor.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/sonukapoor.jpg) (31.3KB)
- [ ] [tracylee.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/tracylee.jpg) (30.6KB)
- [ ] [twerske.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/twerske.jpg) (39.4KB)
- [ ] [wesgrimes.jpg](https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/content/images/bios/wesgrimes.jpg) (463.3KB)

[1]: https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/src/styles/2-modules/_contributor.scss#L152-L153
[2]: https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/ngsw-config.json#L60-L69
[3]: https://github.com/angular/angular/blob/5eb7f3491f37a8f71717525b0d0e901c00a8c4e8/aio/scripts/contributors/validate-data.js#L18-L24

### PR DESCRIPTION
**Pull request by petebacondarwin**
Fixes #41287

### CODE REVIEW

File: aio/scripts/contributors/validate-data.js [hunk 1]
```diff
@@ -44,7 +32,6 @@ function _main() {
 
   // Check that there are no images that exceed the size limit.
   const tooLargeImages = expectedImages
-       .filter(path => !EXCLUDE_FROM_SIZE_CHECK.has(basename(path)))
        .filter(path => statSync(path).size > MAX_IMAGE_SIZE);
```
Comment (gkalpak) [inline aio/scripts/contributors/validate-data.js:35]: Nit:

```suggestion
  const tooLargeImages = expectedImages.filter(path => statSync(path).size > MAX_IMAGE_SIZE);
```

**Review by gkalpak — APPROVED** [review]
Thx for doing this 🤩

A couple of suggestions:
- I think it would be useful to include the method/process/tools you used to reduce the image sizes in the commit message.
- I think it would be a good idea to let the affected contributors know (e.g. by giving them a heads-up in Slack).

**Review by AndrewKushnir — APPROVED** [review]
Thanks for the help @petebacondarwin 👍

**Comment by petebacondarwin** [comment]
> * I think it would be useful to include the method/process/tools you used to reduce the image sizes in the commit message.
Done

> * I think it would be a good idea to let the affected contributors know (e.g. by giving them a heads-up in Slack).
Done

**Review by mgechev — COMMENTED** [review]
Looks like some of the images are still more than 168px:
- aio/content/images/bios/webdave.jpg
- aio/content/images/bios/tracylee.jpg
- aio/content/images/bios/thierrychatel.jpg
- aio/content/images/bios/teriglover.png
- aio/content/images/bios/sonukapoor.jpg
- aio/content/images/bios/sidd-ajmera.jpg
…

**Review by mgechev — APPROVED** [review]

**Comment by petebacondarwin** [comment]
> Looks like some of the images are still more than 168px:
> 
> * aio/content/images/bios/webdave.jpg
> * aio/content/images/bios/tracylee.jpg
> * aio/content/images/bios/thierrychatel.jpg
> * aio/content/images/bios/teriglover.png
> * aio/content/images/bios/sonukapoor.jpg
> * aio/content/images/bios/sidd-ajmera.jpg
>   …

My aim was to get the file-size down. I was not too worried if an image was actually larger than 168px. I think that gives enough flexibility to PR authors who want to add their contributor photos.

**Pull request closed without merging**